wallet/src/ui/app/redux/slices/transactions/index.ts

```diff
@@ -110,7 +110,7 @@ export const StakeTokens = createAsyncThunk<
             amount,
             validatorAddress
         );
-        dispatch(fetchAllOwnedObjects());
+        dispatch(fetchAllOwnedAndRequiredObjects());
```

Comment:
as we now have validator selector, maybe we can replace this placeholder?
https://github.com/MystenLabs/sui/blob/main/wallet/src/ui/app/redux/slices/transactions/index.ts#L99-L100

Reply:
This will be fixed with upcoming PRs that will implement the new design for selecting validators, staking, etc.